config/config.js

```diff
@@ -0,0 +1,30 @@
+import env from "../config/env.generated.json";
+
+let config = {
```

Comment:
small nit, but since this isn't a configuration object calling it `config` is not being true to its use in the code. It's better to call this `userData` or something